translations/monero-core_de.ts

```diff
@@ -140,19 +140,19 @@
     <message>
         <location filename="../components/DaemonConsole.qml" line="108"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Schließen</translation>
     </message>
     <message>
         <location filename="../components/DaemonConsole.qml" line="119"/>
         <source>command + enter (e.g help)</source>
-        <translation type="unfinished"></translation>
+        <translation>Befehl + Enter (z.B. help)</translation>
     </message>
 </context>
 <context>
     <name>DaemonManagerDialog</name>
     <message>
         <source>Daemon doesn&apos;t appear to be running</source>
-        <translation type="vanished">Daemon scheint nicht aktiv zu sein</translation>
+        <translation type="vanished">Dämon scheint nicht aktiv zu sein</translation>
```

Comment:
If you don't have a tech background at all, the english version or quite a lot of things don't make a lot of sense (for example mining). I don't think using a more simplified term helps them a lot, while the more common amount of people would be confused of such a big change. At the moment there is not a lot of information available in German, so people trying to find out about parts of the software will be unable to find anything when googling for "service", as the translation is not obvious. Even if they stumble over someone talking about nodes or daemons in english, he probably won't even know thats the service he is looking for.

So basically:

- Non-Techie people make up a very low amount of actual users and such a slight change won't do much

- Techie people don't care that much but will have a way easier time finding more information about it when we use "Daemon".